app/(stories)/item/[id]/page.tsx

```diff
@@ -85,7 +85,7 @@ export default async function ItemPage({
         <div className="flex-grow">
           {story.url != null ? (
             <a
-              className="text-[#000000] hover:underline"
+              className="text-[#000000] dark:text-[#ffffff] hover:underline"
```

Comment:
Consider using Tailwind CSS utility classes like `text-white` and `text-black` instead of hardcoded color values (#000000 and #ffffff) for improved readability and maintainability.